components/community-groups/CommunityGroupsCard.jsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+  Card, CardContent, Typography, Button, InputLabel, 
+  TextField, FormControl
+} from '@material-ui/core';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import NativeSelect from '@material-ui/core/NativeSelect';
+// EditableCG here
+// import EditableEvent from '../editableEvent/EditableEvent';
+import CommunityGroupModel from '../../src/models/CommunityGroup';

```

Comment:
@jinchizhou the branch doesn't have the CommunityGroups model file in it anymore, but you are still trying to import it here.

Reply:
I have no idea why it compiled before. I forgot to branch off for new features, hence, I'll have to close this PR and create a new one with only this revision.